src/routes/settings/components/SwitchSetting.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import * as React from 'react'
+import { Switch } from 'react-native'
+
+type PropType = {
+  onValueChange: boolean => void,
+  getCurrentValue: () => boolean
+}
+
+type StateType = {
+  value: boolean
```

Comment:
This is a duplication of the state. It should be enough to pass the value as prop.

Reply:
I agree. The only purpose of this component is to "cache" the value. Since this component is not even used anywhere I guess, it can be removed...

Reply:
Yes this is a left-over. Thanks for pointing out!